pkg/controller/glustercluster/csidriver_deployed.go

```diff
@@ -0,0 +1,65 @@
+package glustercluster
+
+import (
+	"fmt"
+
+	"github.com/gluster/anthill/pkg/reconciler"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var glusterFuseProvisionerDeployed = reconciler.NewAction(
+	"glusterFuseProvisionerDeployed",
+	[]*reconciler.Action{},
+	func(request reconcile.Request, client client.Client, scheme *runtime.Scheme) (reconciler.Result, error) {
+		clustername := "clustername"
```

Comment:
name and namespace will come from the `GlusterCluster` object.

Reply:
Yup, this I'm not aiming to merge fully implemented actions in this PR.

Reply:
I'm thinking that we can start working on action implementation in-parallel once this is merged.